ui/components/app/advanced-gas-fee-popover/advanced-gas-fee-inputs/base-fee-input/base-fee-input.js

```diff
@@ -116,7 +127,24 @@ const BaseFeeInput = () => {
 
   useEffect(() => {
     setMaxFeePerGas(maxBaseFeeGWEI);
-  }, [maxBaseFeeGWEI, setMaxFeePerGas]);
+    setMaxBaseFee(maxBaseFeeMultiplier);
```

Comment:
I think default should not be updated as input value change, but rather if input value change we deselect the checkbox and user select it again to update defaults. Updating defaults should happen only from `advanced-gas-fee-defaults.js`.

My understanding is:
- checkbox should be selected if value in inputs is same as defaults
- if user de-select it we remove default
- if user changes input values checkbox is de-delected and as user selects it again defaults are updated

Reply:
This was my initial understanding too, but what if the user assumes the new values will be saved if the default is selected? I will confirm this with the design team.